contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsFetcher.java

```diff
@@ -196,12 +196,17 @@ private Configuration getConfiguration(String sourceFileUrl, String hadoopConfig
 
             String security = config.get(CommonConfigurationKeys.HADOOP_SECURITY_AUTHENTICATION);
 
-            if(security == null || !security.equals("kerberos")) {
-                logger.error("Security isn't turned on in the conf: "
-                             + CommonConfigurationKeys.HADOOP_SECURITY_AUTHENTICATION + " = "
-                             + config.get(CommonConfigurationKeys.HADOOP_SECURITY_AUTHENTICATION));
-                logger.error("Please make sure that the Hadoop config directory path is valid.");
-                throw new VoldemortException("Error in getting Hadoop filesystem. Invalid Hadoop config directory path.");
+            if (security == null) {
```

Comment:
Nitpick : Can we make this a table based method and only warning for the simple ? This code kind of looks little awkward with check for simple and throws if not kerberos, which does the right thing, but little harder to read.

A code like valid options are simple and kerberos. If not throw.
If option equals simple, log a warning is easier to understand.